optuna/pruners/successive_halving.py

```diff
@@ -140,9 +140,7 @@ def _is_promotable(self, rung, value, all_trials, study_direction):
             promotable_idx = 0
 
         if study_direction == StudyDirection.MAXIMIZE:
-            competing_values.reverse()
-            return value >= competing_values[promotable_idx]
-
+            return value >= competing_values[len(competing_values) - 1 - promotable_idx]
```

Comment:
Can we simplify this line as follows?
```suggestion
            return value >= competing_values[-(promotable_idx + 1)]
```

Reply:
Absolutely, applied!